src/routes/User.jsx

```diff
@@ -0,0 +1,95 @@
+// React Imputs
+import React, { useContext, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+// Custom Hook Imports
+import { useStatusNotification } from '../hooks';
+// Utility Imports
+import { getDocTTLs, runNotification } from '../utils';
+// Context Imports
+import { SignedInUserContext } from '../contexts';
+// Component Imports
+import { ShowDocumentsModal } from '../components/DocumentModals';
+import FormSection from '../components/Form/FormSection';
+
+/**
+ * User - Page that displays the User's stored Documents
+ *
+ * @memberof Pages
+ * @name User
+ * @returns {React.ReactElement} The user page
+ */
+
+const User = () => {
+  const location = useLocation();
+  localStorage.setItem('restorePath', location.pathname);
+
+  const { state, dispatch } = useStatusNotification();
+
+  // EXPERIMENTAL - Event handler for displaying Documents from PASS
+  const { session } = useSession();
+  const [fileSrc, setFileSrc] = useState([]);
+  const [showDocument, setShowDocument] = useState(false);
+  const { podUrl } = useContext(SignedInUserContext);
+
+  const handleShowFile = async () => {
+    runNotification('Fetching documents from Pod...', 5, state, dispatch);
+    dispatch({ type: 'SET_PROCESSING' });
+
+    const allDocumentData = await getDocTTLs(session, podUrl);
+
+    if (allDocumentData.message?.includes('No documents found')) {
+      runNotification('Operation failed. Reason: No files found.', 5, state, dispatch);
+      dispatch({ type: 'CLEAR_PROCESSING' });
+      return;
+    }
+
+    const dataWithFiles = allDocumentData.filter((item) => typeof item !== 'number');
+
+    setFileSrc(dataWithFiles);
+    setShowDocument(!showDocument);
+
+    setTimeout(() => {
+      dispatch({ type: 'CLEAR_PROCESSING' });
+    }, 5000);
+  };
+
+  return (
+    <FormSection
```

Comment:
Again, I feel like I'm missing the point (or future point) of this `User` page.  If this is the only thing we don't need it as a form and button combo.  Just run the function and display the documents (like `Clients`.)

Reply:
User would expand, so it'll be fine

Reply:
Temporarily removing "User" until things are more flushed out